scripts/households.py

```diff
@@ -39,35 +38,37 @@ def mu_cons(c, sigma):
 
     return mu_c
 
+
 def mu_labor(n, chi, *args, b_sp1):
     '''
     Use FOC's for disutilitity of labor to get labor supply
-    
-    Refer to eqn 4.9 of ch 4 for the calculation of the 
+
+    Refer to eqn 4.9 of ch 4 for the calculation of the
     marginal utility of labor calculation
-    
+
     n = household labor supply
     chi = utility weight on labor supply
-    
+
     This model solves for the wage rate (w) given the exogeneous nature
     of labor supply
-    
+
     There will be S labor supply Euler equations below
-    
-    ''' 
-    
-    theta = 0.9 # frisch elasticity of labor
-    l_tilde = 1.0 # Maximum labor supply
-    
-    b, nu = elliptical_u_est.estimator(theta, l_tilde)
```

Comment:
@jtregde You will probably want to move this to `execute.py` and pass `b` and `upsilon` from there.  This estimation only needs to be done once - not each time the marginal utility of labor function is evaluated. 

Also, maybe rename `b` to `b_ellip` or something just so that differentiate from `b` for household savings.